lib/pomodoro-timer.js

```diff
@@ -1,6 +1,21 @@
+const readline = require("readline");
+const open = require("open");
+
+//カウントダウン中にctrl+cが押された場合にカーソルを再表示する
+process.on("exit", () => process.stdout.write("\x1b[?25h"));
+process.on("SIGINT", () => process.exit(0));
+
 class PomodoroTimer {
   constructor(settingsJson) {
     this.settingsJson = settingsJson;
+
+    if (process.env.TERM_PROGRAM) {
+      if (process.env.TERM_PROGRAM === "Apple_Terminal")
+        this.terminalProgram = "Terminal";
+      else this.terminalProgram = process.env.TERM_PROGRAM;
+    } else {
+      this.terminalProgram = "";
+    }
```

Comment:
```suggestion
    let terminalProgram = "";
    if (process.env.TERM_PROGRAM) {
      terminalProgram = process.env.TERM_PROGRAM;
    }
    if (terminalProgram === "Apple_Terminal") {
      terminalProgram = "Terminal";
    }
    this.terminalProgram = terminalProgram;
```

とかのほうがわかりやすいかなと思いました。